backend/api/views.py

```diff
@@ -1,3 +1,76 @@
 from django.shortcuts import render
+from django.http import HttpRequest, HttpResponse, JsonResponse
+from django.http import HttpResponseBadRequest, HttpResponseForbidden
+from django.http import HttpResponseNotAllowed, HttpResponseNotFound
+from django.contrib.auth.models import User
+import json
+from json.decoder import JSONDecodeError
 
-# Create your views here.
+
+def HttpResponseOk():
+    return HttpResponse(status=200)
+
+
+def HttpResponseCreated():
```

Comment:
We concluded that we use the status code
* 200 only for success
* ban 401 for failure.

Please change the code to fit in with these policies.